src/python/WMCore/WMSpec/Steps/Template.py

```diff
@@ -62,6 +63,22 @@ def environment(self):
         """
         return self.data.environment
 
+    def setOverrides(self, **overrides):
+        """
+        _setOverrides_
+        set a overrides dictionary
+        """
+        for key, value in viewitems(overrides):
+            setattr(self.data.override, key, value)
+        return   
+
+    def getOverrides(self):
```

Comment:
@germanfgv I think this method is already redefinition of an already existing one:
https://github.com/dmwm/WMCore/blob/0d667606fed0166f304f48c2fc85b9c3b32cc0f9/src/python/WMCore/WMSpec/WMStep.py#L170

it must already be present into the object becuase you inevitably inherit from `WMStep`

Reply:
You are right, I missed this.